RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -196,4 +204,65 @@ public protocol PaywallViewControllerDelegate: AnyObject {
 
 }
 
+// MARK: - Private
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, *)
+private extension PaywallViewController {
+
```

Comment:
we can do it as a separate step, but it is kind of a bummer that because react-native is a pain, all of our hybrids and our UIKit paywalls would suffer. 
might be worth just special-casing this one out

Reply:
> all of our hybrids and our UIKit paywalls would suffer.

What makes you say that?
If you don't use the `update` methods, this is functionally equivalent to the previous implementation.